### PR TITLE
NI state: Handle inactive VIF + minor improvements

### PR DESCRIPTION
**Pull request by milan-zededa**
When application VIF is inactivated, Network Instance State Collector should stop collecting state data for the interface and also it should forget previously recorded IP assignements. Once the interface is reactivated, state collecting is resumed and the newly leased IP address(es) will be recorded and an update will be sent to zedrouter (even if re-activated VIF obtains the same IP as before). This scenario happens for example during an application purge.

This commit additional includes renaming of some variables related to metrics for more clarity.

Lastly, the method collecting ACL counters was incorrectly including the `OUTPUT` chain from the `filter` table, which, however, is not being used and as a result an error message was frequently logged.

### CODE REVIEW

**Review by eriknordmark — APPROVED** [review]
LGTM

File: pkg/pillar/nistate/linux.go [hunk 1]
```diff
@@ -185,49 +225,47 @@ func (lc *LinuxCollector) GetNetworkMetrics() (types.NetworkMetrics, error) {
...
+			TxBytes:  intf.BytesSent,
+			RxBytes:  intf.BytesRecv,
+			TxDrops:  intf.Dropout,
+			RxDrops:  intf.Dropin,
+			TxErrors: intf.Errout,
+			RxErrors: intf.Errin,
 		}
 
-		// Is this interface associated with any network instances?
+		// Is this interface associated with any interfaces instances?
```
[the remark this answers is not in the document]
Comment (milan-zededa) [reply pkg/pillar/nistate/linux.go]: Renaming went too far on this one, should have stayed as `network instance` - fixed.

File: pkg/pillar/nistate/linux.go [hunk 2]
```diff
@@ -185,49 +225,47 @@ func (lc *LinuxCollector) GetNetworkMetrics() (types.NetworkMetrics, error) {
 	// The eethN counters are currently not used/reported, but could be
```
[the remark this answers is not in the document]
Comment (milan-zededa) [reply pkg/pillar/nistate/linux.go:225]: Actually this is explained here: https://github.com/lf-edge/eve/blob/master/pkg/pillar/nistate/linux.go#L182-L187
I took it from the zedrouter and preserved this behavior.

[the remark this answers is not in the document]
Comment (eriknordmark) [reply pkg/pillar/nistate/linux.go:225]: Comment says eeth but don't you mean eth?

[the remark this answers is not in the document]
Comment (milan-zededa) [reply pkg/pillar/nistate/linux.go:225]: No this is intentional and refers to this part of the comment linked above:
> If we have both ethN and kethN then rename ethN to eethN ('e' for EVE)

[the remark this answers is not in the document]
Comment (eriknordmark) [reply pkg/pillar/nistate/linux.go:225]: Ah - got it.

**Pull request merged**